packages/bpk-docs/src/pages/ImagesPage/ImagesPage.js

```diff
@@ -48,7 +50,24 @@ const components = [
         altText="Lake"
         width={816}
         height={544}
-        src={`/${image1}`}
+        src={image1}
+        srcSet={`${image1}?resize=280px:280px&quality=100 280w,
+          ${image1}?resize=320px:320px&quality=100 320w,
```

Comment:
All these examples contain quite a lot of different options - two or three should be adequate, no?

Reply:
Fair point - have reduced to 4